src/Summoner/CLI.hs

```diff
@@ -283,6 +285,9 @@ withoutP = subparser $ mconcat
 ignoreFileP :: Parser Bool
 ignoreFileP = switch $ long "ignore-config" <> help "Ignore configuration file"
 
+noUploadP :: Parser Bool
+noUploadP = switch $ long "no-upload" <> help "Don't upload to github"
```

Comment:
Sorry for pickiness, but as this help text is going to be visible to users I would like to have it pretty, Could you change `github` to `GitHub` instead?

```suggestion
noUploadP = switch $ long "no-upload" <> help "Do not upload to GitHub"
```